cmd/argocd/commands/app.go

```diff
@@ -1099,7 +1101,12 @@ func findandPrintDiff(ctx context.Context, app *argoappv1.Application, proj *arg
 			if !foundDiffs {
 				foundDiffs = true
 			}
-			_ = cli.PrintDiff(item.key.Name, live, target)
+			err = cli.PrintDiff(item.key.Name, live, target)
+			// ignore exitError since diff utilities usually return exit status 1 when diff is present
```

Comment:
Would you mind adding a unit test for this behavior?

Reply:
Sure, let me give it a try.